website/docs/reference/developer-readme.md

```diff
@@ -6,20 +6,27 @@ description: SynapseML Development Setup
 ---
 
 # SynapseML Development Setup
-
+1) Install JDK 11
+    - https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11
+1) On Windows, Configure Hadoop "winutils"
+    - Download from https://github.com/cdarlint/winutils
+    - Create new folder, for exaxmple "C:\Hadoop". Create "C:Hadoop\bin", and place winutils.exe inside
+    - Set winutils.exe properties to "Allow Execution"
```

Comment:
I didnt have to do this

Reply:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/9027725/151848642-34541d38-f564-444c-95b4-c377700e62ad.png">
let me investigate